chain/vm/vm.go

```diff
@@ -166,11 +160,11 @@ type VMOpts struct {
 	Bstore         bstore.Blockstore
 	Syscalls       SyscallBuilder
 	CircSupplyCalc CircSupplyCalculator
-	NtwkVersion    NtwkVersionGetter
+	NtwkVersion    NtwkVersionGetter // TODO: stebalien: In what cases do we actually need this? It seems like even when creating new networks we want to use the 'global'/build-default version getter
```

Comment:
TODO

Reply:
Previously, I needed this to access the state tree. But that shouldn't be necessary now.

Reply:
Oh, wait, I see. The runtime needs it.

Reply:
@magik6k To be clear, the `NtwkVersionGetter` wasn't introduced as part of this PR